### PR TITLE
Hw 3

### PR DESCRIPTION
**Pull request by AlenaKharytonchyk**
Implemented everything according to the task:
image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,5 +47,20 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "^6.5.16",
```
Comment (AliaksandrZahorski) [inline package.json]: it is better to keep versions of installed libs as constant, please remove "^"

File: src/mockedMovies.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        image: "https://picsum.photos/323/486?random=3",
+        title: "Third film",
+        description: "3.Jules Winnfield (Samuel L. Jackson) and Vincent Vega (John Travolta) are two hit men who are out to retrieve a suitcase stolen from their employer, mob boss Marsellus Wallace (Ving Rhames). Wallace has also asked Vincent to take his wife Mia (Uma Thurman) out a few days later when Wallace himself will be out of town. Butch Coolidge (Bruce Willis) is an aging boxer who is paid by Wallace to lose his fight. The lives of these seemingly unrelated people are woven together comprising of a series of funny, bizarre and uncalled-for incidents.—Soumitra",
+        rating: "9,2",
+        genre: ["detective"],
+        year: "2010",
+        duration: "1.4",
+        id: 3,
+    }
+    ,
```
Comment (AliaksandrZahorski) [inline src/mockedMovies.js]: strange comma placement

Comment (AlenaKharytonchyk) [reply src/mockedMovies.js]: Done

File: package.json (hunk 1 above)
Comment (AlenaKharytonchyk) [reply package.json]: Done

**Comment by AliaksandrZahorski** [comment]
Approved

**Pull request merged**